### DISCUSSION TITLE
How can I explicitly set the filename of pkg-config files for package?

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#1637] [Q&A]
例如libpng库，原始的autoconf生成的是libpng16.pc，其他库用pkgconfig找也是找libpng16，但xmake-repo里面用xmake重新port了libpng，而xmake生成的pkg-config文件是libpng.pc，这就导致找libpng16找不到。如何设置xmake中生成的pkg-config文件的名称，或者能否提供方式设置rule utils.install.pkgconfig_importfiles的导出文件名？

**Comment by waruqi** [comment]
现在最简单的方式，就是直接改 port 的 target name 。。`target("png16")`

**Reply by xq114** [reply]
不行，这样生成的文件是png16.pc而不是libpng16.pc，但如果targetname改成libpng16的话生成的库又变成liblibpng.a了

**Reply by waruqi** [reply]
更新下再试试

    add_rules("utils.install.pkgconfig_importfiles", {filename = "libpng16.pc"})
